verifier.py

```diff
@@ -25,7 +28,7 @@
 
 
 # Takes a URL and produces its relative file name.
-def get_path_from_url(self, url):
+def get_path_from_url(url):
     # Remove http://domain
```

Comment:
@adlius We could. I believe you or Sadiyah or Cam gave a reason not to though..?
@sf2ne @cameronblandford